contracts/staking/src/contract.rs

```diff
@@ -193,7 +193,7 @@ pub fn unbond(deps: DepsMut, env: Env, info: MessageInfo, amount: Uint128) -> St
     let owner_raw = deps.api.addr_canonicalize(invest.owner.as_str())?;
 
     // calculate tax and remainer to unbond
-    let tax = amount * invest.exit_tax;
+    let tax = amount.mul_floor(invest.exit_tax);
```

Comment:
This PR is nice. Looking at this diff, I just think we should start working on MIGRATING.md in parallel to making the changes. Otherwise it's going to be a lot to catch up on later. What do you think how to best do this?

Reply:
Good point. We can add them when adding the changelog entry. I can create a PR with a base entry for 1.5 -> 2.0 (with a point about the already merged PR) and then we can add bullet points there.